react-native/setup/setup-firebase.mo.md

```diff
@@ -0,0 +1,140 @@
+
+# [MO] Setup Firebase analytics on your React Native project *(~<Time> min)*
+
+## Owner: [Guillaume LEROY](http://github.com/GuillaumeLe/)
+
+## Prerequisites
+* Have yarn installed
+* Have cocoapods in your project.
+
+## Steps *(~30 min)*
+### Install react-native-firebase *(~2 min)*
```

Comment:
Could you should explain what there in here that isn't in the [official documentation](https://rnfirebase.io/docs/v3.1.*/installation/initial-setup)?

Maybe we should link to the official doc rather than copy pasting so that we keep up to date :)